### ISSUE TITLE
Great presentation!

### ISSUE THREAD

**Issue by Tronic** [wmvanvliet/neuroscience_tutorials#1]
It's good to see someone else also using interactive notebooks to get the point delivered :)

A test export I made https://zi.fi/vis.html (be patient, takes a minute to load). The full notebook with more interactive controls (not only rotation like in the export) and code I have not published because this was a course exercise.

**Comment by wmvanvliet** [comment]
Those plots are beautiful! Super smooth graphics.

How did you make them? Can you share the notebook privately? Or give me the name of the course and teacher?

**Comment by Tronic** [comment]
In every exercise I've tried different visualisation libraries to get the hang of what kind of options there are. There are modules like ipyvolume that even specialise in brain imaging, but yet I couldn't find anything that would provide the output I needed, and thus ended up working directly with pythreejs (which also allows setting camera to use orbitcontrols and avoid the confusing rotation seen in other modules). I also experimented with Plotly for the 2d heatmaps on that one, but found the interface quite bad. (Bokeh is better but still not as convenient as Matplotlib -- and both Plotly and Bokeh struggle at things like setting equal x and y axes).

I sent you an email with the code. Not cleaned up or anything, and has minor bugs, but hopefully that helps. Should you wish to build a vis. module from that or something like that, consider all of the code public domain. The Matlab files (incl. the brain 3d model) are not made by me (ask Matti Stenroos for permission if you wish to use them).

P.S. Plotly, ipywidgets and pythreejs all seem to be somewhat picky on where they are run, and may not work in jupyterlab/colab. Classic notebooks work more reliably.
